app/views/spree/components/_breadcrumbs.html.erb

```diff
@@ -0,0 +1,47 @@
+<%
+  # Props
+  taxon = nil if local_assigns[:taxon].nil?
+  separator = '&nbsp;&raquo;&nbsp;' if local_assigns[:separator].nil?
+  base_class = nil if local_assigns[:base_class].nil?
+
+  # Methods
+  def breadcrumbs(taxon, separator, breadcrumb_class = 'inline')
```

Comment:
A method into a view is definitely something odd. We'll need to revisit it later and look for a better/more standard way to handle this.

Reply:
I thought we already agreed on this approach when we talked about moving logic from the helpers into the component that uses it. We saw that it would be a 1:1 component translation in the context of a future porting (React, Vue). But anyway, I'm open to suggestions.

Reply:
As I said, we can revisit later. For now, I'm fine with this.